### PR TITLE
Sample - analyze and anonymize csv file 

### PR DESCRIPTION
**Pull request by navalev**
Sample for anaylzing and anonymizing csv file - using batch analyzer

### CODE REVIEW

**Review by omri374 — APPROVED** [review]

File: docs/samples/python/example_csv.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    ) -> Iterable[DictAnalyzerResult]:
+
+        with open(csv_full_path, 'r') as csv_file:
+            csv_list = list(csv.reader(csv_file))
+            csv_dict = {header: list(map(str, values)) for header, *values in zip(*csv_list)}
+            analyzer_results = self.analyze_dict(csv_dict, language, keys_to_skip)
+            return list(analyzer_results)
+
+
+class BatchAnonymizerEngine(AnonymizerEngine):
```
Comment (omri374) [inline docs/samples/python/example_csv.py:45]: we should add this to the anonymizer package at a certain point (not for this PR)

Comment (navalev) [reply docs/samples/python/example_csv.py:45]: agree. I'll add it in a seperate PR with supporting unit tests

Comment (omri374) [reply docs/samples/python/example_csv.py:45]: thanks :) happy to help too

File: docs/samples/python/example_csv.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+from presidio_anonymizer.entities import EngineResult
+
+"""
+Example implementing a CSV analyzer
+
+This example shows how to use the Presidio Analyzer and Anonymizer
+to detect and anonymize PII in a CSV file.
+It uses the BatchAnalyzerEngine to analyze the CSV file, and the
+BatchAnonymizerEngine (defined in batch_processing sample) to anonymize
+the requested columns.
```
Comment (omri374) [inline docs/samples/python/example_csv.py:17]: @navalev maybe we should add a link to the batch notebook? Users might get to this sample first, and the other is less technical but provides more explanation into the process. WDYT?

Comment (navalev) [reply docs/samples/python/example_csv.py]: done

File: docs/samples/python/example_csv.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+        :param analyzer_results: Iterator of `DictAnalyzerResult` 
+        containing the output of the AnalyzerEngine.analyze_dict on the input text.
+        :param kwargs: Additional kwargs for the `AnonymizerEngine.anonymize` method
+        """
+
+        return_dict = {}
+        for result in analyzer_results:
+
+            if isinstance(result.value, dict):
```
Comment (shiranr) [inline docs/samples/python/example_csv.py:92]: WHHHHYYYY IS THERE NO SWITCH CASE IN PYTHON??? 😢

**Review by shiranr — APPROVED** [review]

File: docs/samples/python/example_csv.py (hunk 3 above)
Comment (navalev) [reply docs/samples/python/example_csv.py:92]: there's from 3.10 :)

**Review by omri374 — APPROVED** [review]

File: docs/samples/python/example_csv.py (hunk 3 above)
Comment (shiranr) [reply docs/samples/python/example_csv.py:92]: I will create a task for myself to upgrade :D

**Pull request merged**